code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -39,12 +39,19 @@
 	var/charge_time = 1.5 SECONDS
 	var/detonation_damage = 50
 	var/backstab_bonus = 30
+	var/overrides_main = FALSE //bubber edit //do we override the main init?
+	var/overrides_twohandrequired = FALSE //bubber edit //Do we have the fumble on one handed attack attempt?
+	var/override_markeffect = FALSE //bubber edit //Do we have the default affect on detonating a mark?
+	var/override_twohandedsprite = FALSE //bubber edit //ENABLE THIS FOR ALL NEW CRUSHER VARIANTS OR ELSE IT WILL BREAK
+
```

Comment:
These can be made more modular and moved into `modular_zubbers/modules/modular_items/code/miningweapons.dm` plus only two are needed